### PR TITLE
Mlhr 1497 elastic search operators consolidate

### PR DESCRIPTION
**Pull request by yogidevendra**
Incorporating review comments

### CODE REVIEW

**Comment by 243826** [comment]
too old!

File: contrib/pom.xml [hunk 1]
```diff
@@ -218,6 +218,11 @@
       <optional>true</optional>
     </dependency>
     <dependency>
+	    <groupId>org.elasticsearch</groupId>
+	    <artifactId>elasticsearch</artifactId>
+	    <version>1.3.4</version>
+	</dependency>
```
Comment (sandeepdeshmukh) [inline contrib/pom.xml]: Align this properly.

Comment (tweise) [inline contrib/pom.xml]: Ensure dependencies you add to contrib are optional.

File: contrib/src/main/java/com/datatorrent/contrib/elasticsearch/ElasticSearchConnectable.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+ * None<br>
+ * <br>
+ * Run time checks:<br>
+ * hostName batchSize <br>
+ * <b>data type:</br>the insertion data can support all the Objects mongoDB supports<br>
+ *
+ * <b>Benchmarks</b>: <br>
+ *
+ * @since 0.3.2
+ * */
```
Comment (sandeepdeshmukh) [inline contrib/src/main/java/com/datatorrent/contrib/elasticsearch/ElasticSearchConnectable.java]: May need to update @since correctly.

**Pull request closed without merging**